container-engine-lib/lib/backend_impls/docker/object_attributes_provider/docker_label_key/label_key_consts.go

```diff
@@ -37,10 +37,9 @@ const (
 	// We create a duplicate of the enclave uuid and service uuid label key because:
 	// the logs aggregator (vector) needs the enclave uuid and service uuid label keys to create the filepath where logs are stored in persistent volume
 	// but vectors template syntax can't interpret the "com.kurtosistech." prefix, so we can't use the existing label keys
-	logsEnclaveUuidLabelKeyStr         = "enclave_uuid"
-	logsServiceUuidDockerLabelKey      = "service_uuid"
-	logsServiceShortUuidDockerLabelKey = "service_short_uuid"
```

Comment:
service short uuid and service name were no longer needed to create file paths so i removed them, i also renamed these labels to be consistent with the k8s implementation